lib/hasPermissions.ts

```diff
@@ -222,13 +222,23 @@ export class HasPermissionsDirectiveVisitor<
       if (!(missingPermissions && missingPermissions.length > 0)) {
         missingPermissions = null;
       }
-
       if (
         policy === HasPermissionsDirectivePolicy.THROW &&
         missingPermissions
       ) {
         throw new ForbiddenError(getErrorMessage(missingPermissions));
       }
+      /* If existingMissingPermissions is not null it include "policy: RESOLVE" so it's save to continue  */
```

Comment:
i think this comment is just covering one detail

i'd have no comment, if you want at least say "if any missing permissions existed from other hasPermissions that were executed before it, then pass or extend that array with the new permissions".

Something like that.